src/LoginRouter.js

```diff
@@ -198,6 +235,18 @@ function (BaseLoginRouter, PrimaryAuthController, VerifyDuoController, MfaVerify
       });
     },
 
+    enrollWindowsHello: function () {
+      this.render(EnrollWindowsHelloController, {
+        provider: 'FIDO',
+        factorType: 'webauthn',
+        Beacon: FactorBeacon
+      });
+    },
+
+    enrollmentWindowsHello: function () {
+      this.controller.model.activate();
```

Comment:
Also, this doesn't follow the normal model we're using in the rest of the sign-in-widget - once something hits here (we've made a request, we're in a new state), it should render a new controller. I haven't finished looking at the rest of your PR, but if the idea is that it's not hitting a new auth status, then we can trap the response inside the enrollwindowshellocontroller and just call your activate function in there.